lib/config/presets/internal/packages.ts

```diff
@@ -36,18 +36,28 @@ export const presets: Record<string, Preset> = {
     description: 'All Stylelint packages.',
     matchPackagePrefixes: ['stylelint'],
   },
+  standard: {
+    description: 'JavaScript Standard Style',
+    matchPackagePrefixes: ['standard'],
+  },
   tslint: {
     description: 'All TSLint packages.',
     matchPackageNames: ['codelyzer'],
     matchPackagePatterns: ['\\btslint\\b'],
   },
+  prettier: {
+    description: 'Prettier code formatter',
+    matchPackagePrefixes: ['prettier'],
```

Comment:
Same question as above.

Reply:
I think we should use [`matchPackageNames`](https://docs.renovatebot.com/configuration-options/#matchpackagenames) instead, like this:

```suggestion
    matchPackageNames: ['prettier'],
```

This avoids matching `prettier-plugin` or similar names.